pkg/ccl/changefeedccl/resolvedspan/frontier.go

```diff
@@ -6,7 +6,6 @@
 package resolvedspan
 
 import (
-	"context"
```

Comment:
There were some conflicts in this file as a result of 25.1 not having the iterator changes from https://github.com/cockroachdb/cockroach/commit/7e33775827328c90a1b896955b8c071151725163